src/main/java/lotto/domain/LottoNumberGenerator.java

```diff
@@ -0,0 +1,33 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoNumberGenerator {
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private static final int LOTTO_NUMBER_COUNT = 6;
+
+    public static List<Integer> generate() {
+        List<Integer> numbers = createNumbers();
+        Collections.shuffle(numbers);
+        return selectNumbers(numbers);
+    }
+
+    private static List<Integer> createNumbers() {
+        List<Integer> numbers = new ArrayList<>();
+        for (int i = MIN_LOTTO_NUMBER; i <= MAX_LOTTO_NUMBER; i++) {
+            numbers.add(i);
+        }
+        return numbers;
+    }
+
+    private static List<Integer> selectNumbers(List<Integer> numbers) {
+        List<Integer> selectedNumbers = new ArrayList<>();
+        for (int i = 0; i < LOTTO_NUMBER_COUNT; i++) {
+            selectedNumbers.add(numbers.get(i));
+        }
+        return selectedNumbers;
+    }
```

Comment:
```suggestion
    private static List<Integer> selectNumbers(List<Integer> numbers) {
        return new ArrayList(numbers.sublist(0, 6));
    }
```
콜렉션의 sublist 사용해 구현하면 어떨까?

Reply:
콜렉션의 사용을 최대한 지양하고 구현했었는데 stream api를 사용을 지양하자고했지 콜렉션은 상관이없었네요